### PR TITLE
Themes implementation for emojiset

### PR DESCRIPTION
**Pull request by ddemented**
Have implemented the theme selector for emojiset.. Hope you like it 🍸

### CODE REVIEW

**Comment by ddemented** [comment]
sorry i haven't followed standard code style. am new to this, i will try to do this.

File: index.html [hunk 1]
```diff
@@ -259,6 +269,36 @@
...
 
+        <select id="theme" onchange="_change(this);" class ="box">
+            <option selected value="Normal">Normal🐣</option>
+            <option value="Flag">Flag🇺🇸</option>
+            <option value="Fruits">Fruits🍒</option>
+            <option value="Celebration">Celebration🎁</option>
+        </select>
+
+        <script>
+      $(document).ready(function() {
```
Comment (muan) [inline index.html:280]: There is no jQuery on the site so this actually doesn't work. (I'd prefer not adding jQuery here)

File: index.html [hunk 2]
```diff
@@ -259,6 +269,36 @@
         <div class="prepend-input">💣</div>
         <input id="bombs" class="input" type="number" value="10">
 
+        <select id="theme" onchange="_change(this);" class ="box">
```
Comment (muan) [inline index.html:272]: This event should be bound in https://github.com/muan/emoji-minesweeper/blob/gh-pages/game.js#L55 like the others.

File: index.html (hunk 1 above)
Comment (muan) [inline index.html:280]: Actually, #12 has addressed this.

**Comment by muan** [comment]
Stale.

**Pull request closed without merging**